google-cloud-spanner/src/test/java/com/google/cloud/spanner/SpannerOptionsTest.java

```diff
@@ -986,5 +1003,8 @@ public void checkCreatedInstanceWhenGrpcGcpExtensionEnabled() {
     Spanner spanner2 = options1.getService();
 
     assertNotSame(spanner1, spanner2);
+
+    spanner1.close();
```

Comment:
This cleans up errors that are being logged during tests, clogging up the test log.